packages/lib/hooks/useNotifications.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { showToast } from "@calcom/ui";
+
+export const useNotifications = () => {
+  const [buttonToShow, setButtonToShow] = useState<"none" | "allow" | "disable" | "denied">("none");
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Why don’t we use enum instead of hardcoding values? I’ve seen it used in other places as well